docs/data-sources/edge_cluster.md

```diff
@@ -0,0 +1,62 @@
+---
+page_title: "VMware Cloud Foundation Automation: vcfa_edge_cluster"
```

Comment:
I noticed this is the only file which GitHub didn't recognize "the move" of. Might it be some discrepancy here?

Reply:
Yeah, can't find why, it looks good to me